apps/bors_frontend/web/command.ex

```diff
@@ -223,4 +229,10 @@ defmodule BorsNG.Command do
     attemptor = Attemptor.Registry.get(c.project.id)
     Attemptor.tried(attemptor, c.patch.id, arguments)
   end
+  def run(c, {:autocorrect, command}, _) do
+    c.project.repo_xref
+    |> Project.installation_connection(Repo)
+    |> GitHub.post_comment!(
+      c.pr_xref, ~s/Auto-corrected to "#{command}"/)
```

Comment:
With this change there will be a msg in the pr saying:
"It's running because it was autocorrected"

I was leaning towards not executing anything at all when there is a typo and letting the user know. The same way git works:
```
$ git statys
git: 'statys' is not a git command. See 'git --help'.

Did you mean this?
	status
```

Making the typo trigger an action is just incrementing the commands inside bors. If you add this, someone might come in the future and would also like to have: bor+, ++, etc.